Src/static/styles.css

```
@@ -1,55 +1,173 @@
+  color: var(--text-primary);
+}
+
+/* ===========================================
+   BUTTONS
+   =========================================== */
+.btn-glass {
+  border-color: rgba(255, 255, 255, 0.3);
+  color: rgba(255, 255, 255, 0.9);
+  transition: var(--transition);
+  background: transparent;
+  border: 1px solid;
```

Biome (2.1.2)

[error] Unexpected shorthand property border after border-color



(lint/suspicious/noShorthandPropertyOverrides)